### DISCUSSION TITLE
calc 2.14.1.0 release

### DISCUSSION THREAD

**Discussion by lcn2** [lcn2/calc#55] [Announcements]
The following are the changes in this release:

    Fixed alignment of show item section of help command.

    Bug fix: While 0^0 == 1, now for y > 0, 0^y == 0.
    Adjusted cal/test8900.cal to reflect this bug fix.
    Added tests to cal/regress.cal to help verify bug fix is fixed.

<hr /><em>This discussion was created from the release <a href='https://github.com/lcn2/calc/releases/tag/v2.14.1.0'>calc 2.14.1.0 release</a>.</em>

**Comment by BilakshanP** [comment]
Keep it up dude :heart:

**Comment by lcn2** [comment]
Thank you kindly @BilakshanP
